src/spec.c

```diff
@@ -2772,12 +2772,12 @@ SpecOpIndexingCtx *Indexes_FindMatchingSchemaRules(RedisModuleCtx *ctx, RedisMod
   dict *specs = res->specs;
 
 #if defined(_DEBUG) && 0
-  RLookupKey *k = RLookup_GetKey_TEMP(&r->lk, UNDERSCORE_KEY, RLOOKUP_F_NOFLAGS);
+  RLookupKey *k = RLookup_GetKey_Load(&r->lk, UNDERSCORE_KEY, UNDERSCORE_KEY, RLOOKUP_F_NOFLAGS);
```

Comment:
the difference between `RLookup_GetKey_Load` and `RLookup_GetKey_LoadEx` is the `name_len` field. We don't have it so it will be get internally. maybe calling it directly with `strlen(UNDERSCORE_KEY)` will be optimized by the compiler though.
On another note, this code is commented out. maybe we can just remove it?